Makefile

```
@@ -5,6 +5,17 @@ CARGO = cargo
 # Let's be very explicit about it for now.
 TESTS_DIR := `pwd`/tests
 TEST_RESULTS_DIR ?= workspace/test-results
+
+# In order to be consumed by the ETE Test Metric Pipeline, files need to follow a strict naming
+# convention: {job_number}__{utc_epoch_datetime}__{workflow}__{test_suite}__results{-index}.xml
+WORKFLOW := build-test-deploy
+EPOCH_TIME := $(shell date +%s)
```

Might want to quote the formatting string. It's good hygene 
```suggestion
EPOCH_TIME := $(shell date +"%s")
```